### ISSUE TITLE
ELI5 dataset

### ISSUE THREAD

**Issue by ChantalMP** [plageon/HtmlRAG#1]
Hi :)

I wanted to ask if you have access to the supporting documents in the ELI5 dataset and if you could provide some information on where to find them.

Thanks and best regards,

Chantal

**Comment by plageon** [comment]
We will release the retrieved HTML from Bing soon.

**Comment by plageon** [comment]
The training and test data are now available in the huggingface dataset [HtmlRAG-train](https://huggingface.co/datasets/zstanjj/HtmlRAG-train) and [HtmlRAG-test](https://huggingface.co/datasets/zstanjj/HtmlRAG-test).
By the way, full ELI5 dataset can be accessed at [FlashRAG_datasets](https://huggingface.co/datasets/RUC-NLPIR/FlashRAG_datasets).
Hope this solves your concern. I close this issue for now.
